std/array.d

```diff
@@ -3339,6 +3340,69 @@ if (isDynamicArray!A)
     app.put(r[]);
 }
 
+unittest // Issue 13300
+{ 
+    // Issue 13300: appender must work with non pure postblit
+    import std.array : array;
+
+    struct Simple13300
+    {
+        @disable this(); 
+        this(this) { import std.stdio : stdout; stdout; }  // Not pure!
```

Comment:
Please just use locally declared function that's not `pure` rather than importing std.stdio. Otherwise, you're impacting compile times for little benefit by importing a module that you don't need.

Reply:
I don't know how to modify global state from inside a unittest block in a trivial way. Probably I'm missing something.